src/h5web/toolbar/controls/DomainSlider/DomainTooltip.tsx

```diff
@@ -47,6 +54,8 @@ const DomainTooltip = forwardRef<Handle, Props>((props, ref) => {
   const minEditorRef = useRef<BoundEditorHandle>(null);
   const maxEditorRef = useRef<BoundEditorHandle>(null);
 
+  const [controlsRef, controlsSize] = useMeasure<HTMLDivElement>();
```

Comment:
I measure the size of the `div` containg the controls to give the same size to the histogram. That gives a visually pleasing ratio.